### ISSUE TITLE
Dual SIM - using wrong SIM

### ISSUE THREAD

**Issue by AlexOnFOSS** [mgolden/kdecsms#4]
Hi mgolden,

I just tested your kdecsms and unfortunately I have an issue.
I have a phone with two SIM-Cards in it. When Sending an SMS it is always using the SIM-Card from my first SIM-Slot. I cannot express that I want to send the SMS with the SIM from the second SIM-Slot.

On the other hand, using the kdeconnect built in application to answer a SMS it will use the SIM-Card (slot one or slot two) that had received the message. 

In other words:
When using your application to send a SMS from phone number in slot 1 to phone number in slot 2 everything works fine.
When using your application to send a SMS from phone number in slot 2 to phone number in slot 1 it does not work.
When using the kdeconnect popup to answer to a received SMS from phone number in slot 2 to whatever phone number it will use slot 2 as the sender which is intended.

**Comment by mgolden** [comment]
Thanks for the bug report, and I am happy to hear you (are trying to) use it. I would be happy to fix the issue if we can figure out how.

I don't have a phone with two sims, so I am not sure how to test it. The whole program is really just a wrapper around the command-line program kdeconnect-sms. When I look at the options for that, I get this:

$ kdeconnect-sms --help
Usage: kdeconnect-sms [options]
SMS Instant Messaging

Options:
  -h, --help                 Displays help on commandline options.
  --help-all                 Displays help including Qt specific options.
  -v, --version              Displays version information.
  --author                   Show author information.
  --license                  Show license information.
  --desktopfile <file name>  The base file name of the desktop entry for this
                             application.
  --device <id>              Select a device
  --message <message>        Send a message

The way to find out what devices there are is:

$ kdeconnect-cli -l
- Robin: 0...b (paired)
- Galaxy Nexus: a...e (paired)
- Moto G(7): f..b (paired and reachable)
3 devices found

Does your phone show up as two different devices? How do you tell it which phone number to use when using kde-connect?

**Comment by AlexOnFOSS** [comment]
kdeconnect-cli -l
- Mi A3: ... (paired and reachable)
1 devices found

I don't know how kdeconnect's reply function works under the hood. I guess that function might be the right starting point for this.

I have attached two files to this message in the hope it makes things more clear.
The first picture show the notification i receive on incoming SMS. After clicking reply (Antworten) the second window (pic 2) pops up. 
The second picture shows the reply window that allows me to answer to the SMS with the same Sim-Card / Number that the SMS was received. 
![sms-notify](https://user-images.githubusercontent.com/49358870/119615579-d99a2480-bdff-11eb-962f-3b03da2e42b9.png)
![sms-reply](https://user-images.githubusercontent.com/49358870/119615583-da32bb00-bdff-11eb-9b80-b22ffe613eb4.png)
